bpmn-model/pom.xml

```diff
@@ -57,6 +59,16 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>clirr-maven-plugin</artifactId>
+        <configuration>
+          <ignored>
+            <ignored>
+              <!-- ignore new methods in the Zeebe element instance interfaces -->
```

Comment:
Why explicitly only this interface? As I understand it we want to be backwards compatible, so we should change or delete existing interfaces, but we should always be able to add new ones right?

Reply:
Adding new methods should be okay. I included only the package `io/zeebe/model/bpmn/instance/zeebe` because that was the scope that I changed. In the future, we may include other packages. But for now, I would limit it to what we need.